feedthefox/users/adapters.py

```diff
@@ -92,15 +123,31 @@ def pre_social_login(self, request, sociallogin):
             user = emails_query[0].user
 
         if user:
+            # If this is a new social account, connect it please
             sociallogin.user = user
+
+            # Add additional emails, if any
+            addresses, primary = cleanup_email_addresses(request, sociallogin.email_addresses)
+
+            for address in addresses:
+                address.user = user
+                address.save()
+            if not primary:
+                primary_emails = [addr for addr in sociallogin.email_addresses
+                                  if addr.primary]
+                if primary_emails:
+                    # The last is primary
```

Comment:
Both are primaries. Please change the comment to ``The last is the primary email from mozillians.org``.